conmon-rs/server/src/container_io.rs

```diff
@@ -104,11 +110,24 @@ impl ContainerIO {
         pipe: Pipe,
         logger: SharedContainerLog,
         message_tx: UnboundedSender<Message>,
+        attach: SharedContainerAttach,
     ) -> Result<()> {
         let stream = unsafe { File::from_raw_fd(fd) };
+        let mut writer = unsafe { File::from_raw_fd(fd) };
         let mut reader = BufReader::new(stream);
         let mut buf = vec![0; 1024];
         loop {
+            if let Some(data) = attach
```

Comment:
we should register a new routine for stdin processing when we register the read loops IMO

Reply:
Absolutely, I'll create a dedicated task for that :+1: